### ISSUE TITLE
Comfee »Smartcool 7000 Wifi (midea MPPH-07CRN7) .. possible ??

### ISSUE THREAD

**Issue by The-Paw** [ttimpe/homebridge-midea#55]
Works it with this PlugIn ??

**Comment by luggi83** [comment]
Since the app registration for that device uses the midea smarthome app (https://www.youtube.com/watch?v=9JVtma3XUp8) I'd tentatively say yes but I'll be able to tell you for sure tmr or day after!

**Comment by luggi83** [comment]
Yep, works. The trick is, don't use the Msmarthome app, but tjhe Midea air app. Register the device as portable ac (after pressing the mode key for three seconds until you see AP in the display) and it works automagically!

**Comment by ddbkva** [comment]
thank you
